README.md

```diff
@@ -21,6 +21,7 @@ Then `bundle install`.
 Here's a quick example, adding the middleware to a Rails app in `config/initializers/omniauth.rb`:
 
 ```ruby
+
```

Comment:
What I meant was the Github pages site is out of sync with the readme.
Sorry for the confusion.

On Jul 18, 2016 8:09 AM, "Dylan Thacker-Smith" <notifications@github.com>
wrote:

> In README.md
> <https://github.com/Shopify/omniauth-shopify-oauth2/pull/44#discussion_r71153154>
> :
>
> > @@ -21,6 +21,7 @@ Then `bundle install`.
> >  Here's a quick example, adding the middleware to a Rails app in `config/initializers/omniauth.rb`:
> >
> >  ```ruby
> > +
>
> This looks unnecessary
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Shopify/omniauth-shopify-oauth2/pull/44/files/dc9690df5951685acc5704bf9640f2d14f54e26e#r71153154>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACWjqWQL0NTv7QKyB84CP4aLReLq3lNUks5qW4kegaJpZM4JOHhK>
> .
>